lib/services/import_user.rb

```diff
@@ -0,0 +1,120 @@
+# The class encapsulates the following services:
+#   - adding new users into DB,
+#   - updating users in DB.
+#
+# A hash of the following attributes is required for a new instance:
+#   - first_name
+#   - last_name
+#   - email
+#   - birth_date
+#   - employment_date
+#   - password
+# The hash attributes are mapped into appropriate original fields.
+# For example:
+# ```
+# 'first_name' => 'Name'
+# ```
+#
+# The service tries to create a record in DB.
+# There following outcomes are possible:
+#   - new user is added into DB,
+#   - existing user is updated with provided data,
+#   - fail due to errors.
+#
+# A new user is added into DB if there was no error during record
+# creation DB transaction.
+#
+# A record creation DB transaction can fail. An error with
+# the 'Email has already been taken' message is possible.
+# In this case the existing user with given email is updated with provided data.
+#
+# In case of any other errors the service just reports about errors without
+# any changes in DB.
+class ImportUser
+  EXPECTED_COLUMNS = {
+    'first_name' => "Ім'я",
+    'last_name'  => 'Прізвище',
+    'email'      => 'Email',
+    'birth_date' => 'ДН',
+    'password'   => 'password',
+    'employment_date' => 'Дата приходу на роботу'
```

Comment:
Won't fix.
I find it not reasonable to implement a whole feature for a simple rake task.